mmseg/models/segmentors/base.py

```diff
@@ -107,6 +107,16 @@ def forward(self, img, img_metas, return_loss=True, **kwargs):
         if return_loss:
             return self.forward_train(img, img_metas, **kwargs)
         else:
+            if torch.onnx.is_in_onnx_export() or kwargs.pop('do_norm', False):
+                assert len(img_metas) == 1
+                img_norm_cfg = img_metas[0][0].get('img_norm_cfg', None)
+                if img_norm_cfg and img_norm_cfg['normalize_in_graph']:
+                    mean = torch.tensor(img_norm_cfg['mean'])[None, ..., None,
+                                                              None]
+                    std = torch.tensor(img_norm_cfg['std'])[None, ..., None,
+                                                            None]
+                    img = [(i - mean) / std for i in img]
```

Comment:
Since dynamic axis on batch dim is supported in pytorch2onnx in mmseg, could we do normalization with batch dim?

Reply:
I'm sorry I don't quite understand what change you're proposing. I do know that the resulting model after normalizing in graph with `--normalize-in-graph` and `--dynamic-export` set does have a dynamic batch axis (assuming `cfg.model.test_cfg.mode == 'whole'`).

Reply:
What I'm trying to say is that have you tested batch inference with batch_size>1 when set both `--normalize-in-graph` and `--dynamic-export` and if it is OK?

Reply:
Ah, good point. Thanks for bringing it up. I just checked and ensured that batch inference with batch_size > 1 does work with models exported using `--normalize-in-graph` and `--dynamic-export`.